_posts/2017-07-20-minority-report.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: >
+  Minority report
+author: riikkanen
+excerpt: The pictures used in recruitment annoucments show always a partial glimpse of every day life in Solita. A little more in this minority report.
+tags:
+- culture
+- recruitment
+---
+
+We got feedback that the pictures used in the recruitment announcements give a little biased picture of typical employee at Solita. They show mostly about 30 year old white males. Well, that's true, because I think at least the majority of developers in Solita are about 30 year old white males. Of course, in reality there are plenty of us, some male, some female, all in different bodies. The pictures in the advertisements are not a statement what a typical developer should look like. What really matter are the skills, personality and how Solita-like you are. ([One true story](http://dev.solita.fi/2017/04/07/solita-java-trainee.html))
+
+## Real People - No Photostock
+
+In my opinion it is great that Solita uses authentic pictures of real employees in these advertisements. Some of the pictures are even shooted in real situations during the work day. At least the one I'm in presents a real case where we were debugging the software with a colleague when the photographer just came out with his camera. I think that picture is used e.g. for recruiting integration specialists. It must be remembered, that the picture chosen in the announcement is always a partial glimpse to the day in life in Solita. I'm not an integration specialist and we were not recruiting especially women. 
```

Comment:
itse asiassa sen pitäis olla _taken_. "Shoot" taitaa viitata yleensä videokuvaan.